docs/user/reference/openapi.yaml

```diff
@@ -357,25 +356,20 @@ paths:
           description: Successful Response
       summary: Get State
     put:
-      description: 'Request that the worker is put into a particular state.
-
-        Returns the state of the worker at the end of the call.
-
-        If the worker is PAUSED, new_state may be RUNNING to resume.
-
-        If the worker is RUNNING, new_state may be PAUSED to pause and
-
-        defer may be True to defer the pause until the new checkpoint;
-
-        STOPPING to stop execution of the current task, marking the
-
-        current run as a Success; ABORTING to stop execution of the
-
-        current task, marking the current run as Aborted, reason may be
-
-        a string reason for why the run is to be aborted.
-
-        All other values of new_state will result in 400 "Bad Request"'
+      description: "Request that the worker is put into a particular state.\nReturns\
```

Comment:
Doesn't one of the many ways of writing strings in yaml let you avoid having to insert literal `\n`s for line breaks?

Reply:
Closing as we discussed, it's just the autogenerated schema